client/internal/engine.go

```diff
@@ -353,6 +353,15 @@ func signalCandidate(candidate ice.Candidate, myKey wgtypes.Key, remoteKey wgtyp
 	return nil
 }
 
+func sendSignal(message *sProto.Message, s signal.Client) error {
+	err := s.Send(message)
```

Comment:
I think easier if you return with s.Send(message), without check the err inside this scope